features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreFragment.kt

```diff
@@ -58,5 +59,10 @@ class ExploreFragment : Fragment(R.layout.fragment_explore) {
             viewModel.logEvent(AnalyticsConstants.CrowdNode.STAKING_ENTRY)
             viewModel.openStaking()
         }
+
+        binding.stakingApy.text = getString(
+            R.string.explore_staking_current_apy,
+            String.format(Locale.getDefault(), "%.1f", viewModel.getCrowdNodeAPY())
```

Comment:
Do we need to specify the default local to make sure that the number is rendered as 5.9 or 5,9?

Reply:
QA says this works.